CommunityGuidelines.md

```diff
@@ -0,0 +1,61 @@
+# Community Guidelines
+
+## Community rules
+
+- Please be respectful to other members. Avoid any racist, sexist, homophobic, excessively political, or otherwise distasteful discussions.
+- While some cursing is appropriate in context, refrain from using excessive profanity.
+- Keep conversations in the appropriate channel. Some off-topic discussions are to be expected but try not to blatantly post non-subject-related content.
+- No NSFW material is allowed. Breaking this rule will be met with an immediate ban.
+- Keep spam to a minimum:
+  - Do not repeatedly send the same message.
+  - Do not post links to sites/content others would find unacceptable.
+- Promoting your content, creations, YouTube channel, etc. is encouraged, but you must have permission from staff to advertise another Discord community.
+- Advertising or promotion of piracy is forbidden.
+- Staff members (`@Developer` on Discord) have the final say about what is and is not acceptable within the community. Refusals to listen to staff may result in being kicked or banned.
+- HAVE FUN and enjoy all that the community has to offer.
+
+To read our Code of Conduct please refer to `CODE_OF_CONDUCT.md`.
```

Comment:
```suggestion
To read our Code of Conduct please refer to [Code of Conduct](CODE_OF_CONDUCT.md).
```

Technically relative hyperlinks work. If they do not, we can add the permalink, but I think using hyperlinks is recommended here.